.circleci/config.yml

```diff
@@ -464,6 +464,31 @@ jobs:
         command: |
           pact-broker can-i-deploy --broker-base-url "https://pact-broker.dsp-eng-tools.broadinstitute.org/" --pacticipant aou-rwb-api --version ${CIRCLE_SHA1} --to-environment dev --retry-while-unknown 3 --retry-interval 30
 
+  #Updates the pacticipant version in the pact broker for our test environment which is referred to as dev in Pact Broker
+  api-contract-update-test:
+    executor: pact-executor
+    steps:
+    - run:
+        name: Pact Broker, update test.
+        working_directory: ~
+        command: |
+          pact-broker record-deployment --broker-base-url "https://pact-broker.dsp-eng-tools.broadinstitute.org/" --pacticipant aou-rwb-api --version ${CIRCLE_SHA1} --environment dev
```

Comment:
Pact Broker is only aware of three environments (dev, staging, and production). Our test environment is equivalent to Pact Broker's dev, so that is why you see `--environment dev` here.